### DISCUSSION TITLE
My GitHub Education submission keeps getting dismissed

### DISCUSSION THREAD

**Discussion by Kaimeliax** [community/community#146774] [GitHub Education]
### Hi everyone,

Hello,
I am currently going to Taurages PRC school, i got an old style issued school ID, and i am currently on Erasmus+ trip for 3 weeks with school, i did add the letter which had info about the trip, but still did not pass, i tried GPS emulator and used Telia LT roaming data, still not passing, and my school name is not Taurages vocational, its profesion training center because they get you ready for a profesion and you get a degree for that profesion.
Fun fact i aplied for Viada Study card (which gives students discounts for fuel and etc.) and it passed from first time

**Comment by MindfulLearner** [comment]
Hey try to contact the github https://support.github.com/tickets/personal/0

**Comment by Kaimeliax** [comment]
I did add a document from my erasmus+ trip, as im on it right now, how tf i
can avoid roaming if im in italy on a erasmus+ training trip with school,
and no neither my or other schools issue a digital ID, lithuania is far
from that technology, and yes, i googled it should be vocational not
profession training school, so github has correct info

**Comment by Kaimeliax** [comment]
And btw, viada uses out NŠA database to see if you are a real student, they
enter ur name and surname and then it shows up, its much simples system, i
mean you can look at my school id, its valid id, because its older style
doesnt mean its not valid 😄
